packages/pilot/src/containers/Reprocess/Confirmation/style.css

```diff
@@ -0,0 +1,10 @@
+@import "former-kit-skin-pagarme/dist/styles/spacing.css";
+
+.title {
+  text-align: center;
+  padding-bottom: var(--spacing-small);
```

Comment:
Para seguir melhor exemplo o zeplin 
![image](https://user-images.githubusercontent.com/1773766/65889153-1ede2880-e377-11e9-94a8-554f8cb171a3.png)
Será necessário adicionar 16px no topo do título também:
```suggestion
  padding: var(--spacing-small) 0;
```

Reply:
Resolvido!